students/Ruohan/lesson07/html_render.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element:
+    tag = 'html'
+    def __init__(self, content=None, style="line-height:200%"):
```

Comment:
I think we went over this last night together in person, but this PR doesn't seem to include your latest changes. An element can have any number of attributes, so use `**kwargs`, save the dict properly and render it in the opening tag.